_posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "A Report about Presto Conference Tokyo 2020 Online"
+author: Toru Takahashi, Treasure Data
+excerpt_separator: <!--more-->
+---
+
+On Nov 11th, 2020, Japan Presto Community has held the 2nd Presto Conference 
+with welcoming Martin Traverso and Brian.
+The conference was hosted at Youtube Live.
+This article is the summary of the conference aiming to share their great talks. 
+
+<!--more-->
+
+# Presto Community Updates
+
+First of all, Martin introduce recent Presto updates in these days. 
+It covers rencent changes and enhancements achieved by the community activities.
+Attendees also learned several new functions that will be available soon.
+
+- Update / Merge (https://github.com/prestosql/presto/issues/3325)
+- Materialized Views (https://github.com/prestosql/presto/pull/3283)
+- Dynamically resolved functions
+- Optimized Parquet reader
+
+In addition, at Q&A, he suggests new developers who want to contribute to PrestoSQL 
+to check "good first issue" tag on Github. The tag is a good first step for a new joiner to contribute. 
+Ref. [link](https://github.com/prestosql/presto/labels/good%20first%20issue)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=189" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+# Presto Community - How to get involved
+
+To make attendees get used to Presto Community, Martin provided a guide for walking around Presto community. 
+He gives us their team's principles about the Presto community, and talk about their education strategy for new Presto users.
+I would like to quote the pricinpals here.
+
+- We are passionate about open source
+- We help others be succesfful with what we create
+- We create robust long-lasting software
+- We are egalitarian (nobody is more important than the other)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=1429" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
youtube tag

Reply:
@mosabua Does youtube tag support start-time option?

Reply:
Since their talks are in series, let me remove this one.